controllers/azuremanagedcontrolplane_controller.go

```diff
@@ -237,8 +237,7 @@ func (amcpr *AzureManagedControlPlaneReconciler) reconcileNormal(ctx context.Con
 			return reconcile.Result{}, err
 		}
 	}
-
-	svc, err := newAzureManagedControlPlaneReconciler(scope)
+	svc, err := getNewAzureManagedControlPlaneReconciler(scope)
```

Comment:
nit: preserve the newline unless it's better style-wise

Reply:
This may have been an editor thing.  I'll remove the newline.

Reply:
Removed and resolving.